### DISCUSSION TITLE
Error when User::factory()->create() on ModelObserver::saving($model) : model must be of type CipherSweetEncrypted, User given

### DISCUSSION THREAD

**Discussion by petergithubmgw** [spatie/laravel-ciphersweet#34] [Q&A]
Ciphersweet working okay creating users and login, but I can't test as User::factory()->create() throws next error:
Spatie\LaravelCipherSweet\Observers\ModelObserver::saving(): Argument #1 ($model) must be of type Spatie\LaravelCipherSweet\Contracts\CipherSweetEncrypted, App\Models\User given, called in ...\vendor\laravel\framework\src\Illuminate\Events\Dispatcher.php on line 441
I've tried adding use UsesCipherSweet; at UserFactory class (I've added configureCipherSweet function too), but it keeps failing.
What I'm missing? Thanks.

**Comment by olgabast** [comment]
Got the same error, you need to add `implements CipherSweetEncrypted` to the encrypted model (and read the docs more carefully ^_^)

**Reply by petergithubmgw** [reply]
Too bad my class was like this already:
class User extends Autentificable implements CipherSweetEncrypted
